platform/android/LICENSE.md

```diff
@@ -1,128 +1,5 @@
 <!-- This file was generated. Use `make android-license` to update. -->  
 ## Additional Mapbox GL licenses  
-Mapbox GL uses portions of the Android Arch-Common.  
```

Comment:
This doesn't seem right :)

Reply:
on master I'm getting the same diff...

Reply:
woops, I was not getting the same diff :upside_down_face: 
should be fixed with https://github.com/mapbox/mapbox-gl-native/pull/14792/commits/3929c6f2d47a1b60933a4206edc2b40c1f5f2e04